packages/docs/src/util/helpers.js

```diff
@@ -40,29 +40,32 @@ export function getComponent (type) {
 }
 
 export function parseLink (match, text, link) {
-  let attrs = ''
-  let icon = ''
-  let linkClass = 'v-markdown--link'
-
-  // External link
-  if (
+  const isInternal = !(
     link.indexOf('http') > -1 ||
     link.indexOf('mailto') > -1
-  ) {
-    attrs = `target="_blank" rel="noopener"`
-    icon = 'open-in-new'
-    linkClass += ' v-markdown--external'
-  // Same page internal link
-  } else if (link.charAt(0) === '#') {
-    icon = 'pound'
-    linkClass += ' v-markdown--same-internal'
-  // Different page internal link
-  } else {
-    icon = 'page-next'
-    linkClass += ' v-markdown--internal'
+  )
+  const isSamePage = link.startsWith('#')
+
+  const attrs = isInternal ? '' : `target="_blank" rel="noopener"`
+  const classes = 'v-markdown--link'
+  const icon = isInternal ? 'page-next' : 'open-in-new'
+  const [url = '', hash = ''] = link.split('#')
+
+  if (isInternal && !isSamePage) {
+    const lang = window.localStorage.getItem('currentLanguage') || 'en'
```

Comment:
rendered routes on build render with the following `window is not defined` error
![123123123](https://user-images.githubusercontent.com/27827979/76264481-54494500-6238-11ea-8591-30a096ded4d4.png)

Reply:
Resolved